src/OpenSage.Game/Game.cs

```diff
@@ -578,6 +578,11 @@ public void ShowMainMenu()
 
                         players[i].SelectUnits(new [] { startingBuilding });
                     }
+
+                    if (players[i].IsHuman)
```

Comment:
We shouldn't set this inside of here. Replays have no local player

Reply:
We already set the local player index in this method, so I think this is okay. This change makes it a little better, but you're right that ultimately this isn't the right place to do it.